### PR TITLE
Remove Envoy messages 🐿 v2.12.5

### PR DESCRIPTION
**Pull request by saldixon**
Removal of 28 messages which were used by Envoy (as per https://financialtimes.atlassian.net/browse/IDS-3):

appPromoMobile
b2bTrialAnon
b2bTrialContactUs
b2bTrialCountdown
b2bTrialMember
b2bTrialMobile
b2bTrialMyFt
b2bTrialNewsletter
b2cSubAcquisitions
b2cTrialAcquisitions
bookYourConsult
coronavirusNewsletterPromo
dailyDigest
fastFt
ftWeekendPromo
giftArticles
marketsData
myFtAccountTopicRecs
myFtFeedpage
myFtFeedpageOverview
nbeAutoSub
newsletterPromo
nextBestAction
onboardingMyFt
onboardingPremium
onboardingRegion
oneClickDailyDigest
techScrollAsia

### CODE REVIEW

File: manifest.js [hunk 1]
```diff
@@ -51,116 +51,16 @@ module.exports = {
...
 		path: 'bottom/us-election-2020-promo'
 	},
-	coronavirusNewsletterPromo: {
-		path: 'bottom/coronavirus-newsletter-promo'
-	},
-	nextBestAction: {
-		path: 'top/next-best-action',
-		lazy: true
-	},
 	coronavirusNewsletterPromoTop: {
```
Comment (thurstontye) [inline manifest.js:64]: Looks like ` client/components/bottom/coronavirus-newsletter-promo/main.scss ` has been removed. So does this need to come out of the manifest or the style deletion reverted.

File: main.scss [hunk 2]
```diff
@@ -2,21 +2,7 @@ $system-code: 'n-messaging-client';
 
 @import './client/themes';
 @import './client/components/top/anon-subscribe-now/main';
-@import './client/components/bottom/b2b-trial-myft/main';
```
Comment (thurstontye) [inline main.scss:5]: Are these all solely driven by Envoy? It looks like in next-ammit at least `coronavirusNewsletterPromoTop` is in the messaging.json with no reference to envoy.

File: manifest.js (hunk 1 above)
Comment (saldixon) [reply manifest.js:64]: that stylesheet was only used by the coronavirusNewsletterPromo message (deleted in this PR).  The remaining coronavirusNewsletterPromoTop message doesn't rely on its own bespoke stylesheet .. instead uses n-message* and o-message* classes to pick up styles.

File: main.scss (hunk 2 above)
Comment (saldixon) [reply main.scss:5]: The deleted ones are all as per instructions here:   https://financialtimes.atlassian.net/browse/IDS-3

coronavirusNewsletterPromoTop has never been controlled by Envoy.  (unlike coronavirusNewsletterPromo which was).

**Comment by saldixon** [comment]
> Not 100% sure all of these are envoy messages.

Is it just the coronaVirus one(s) you're concerned about ?  It's confusing because there were two versions .. one was envoy controlled and has been removed, the other remains.  (controlled purely by Ammit)

File: manifest.js (hunk 1 above)
Comment (thurstontye) [reply manifest.js:64]: ok

File: main.scss (hunk 2 above)
Comment (thurstontye) [reply main.scss:5]: Ok names too similar!

**Pull request merged**